app/services/teachers/school_transfers/history.rb

```diff
@@ -0,0 +1,72 @@
+module Teachers::SchoolTransfers
+  class History
+    ALLOWED_TEACHER_TYPES = %i[ect mentor].freeze
+
+    Transfer = Data.define(:leaving_training_period, :joining_training_period) do
+      def type
+        return :unknown unless joining_training_period
+        return :new_provider if leaving_training_period.lead_provider !=
+          joining_training_period.lead_provider
+
+        :new_school
```

Comment:
Hi @cpjmcquillan maybe we need to check if the school is different before assuming it's a new_school?

Reply:
I'm not sure we need to check if the schools are different, its covered already when building the transfers?

https://github.com/DFE-Digital/register-early-career-teachers-public/pull/1724/files#diff-6071826e221eb1062a0f0f283919315891c8ab28f5a39b90ae1f50160ec93a27R14